### PR TITLE
Add lamarzocco documentation

### PR DESCRIPTION
**Pull request by zweckj**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/102291
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4798
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/lamarzocco.markdown [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ha_domain: lamarzocco
+ha_platforms:
+  - water_heater
+ha_codeowners:
+  - '@zweckj'
+  - '@rccoleman'
+ha_integration_type: integration
+---
+
+This integration interacts with [La Marzocco coffee machines]() through a mix of cloud calls, local API calls, WebSocket connections and (optionally) Bluetooth.
```
Comment (c0ffeeca7) [inline source/_integrations/lamarzocco.markdown]: Did you have a specific, external link in mind here?

File: source/_integrations/lamarzocco.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ha_domain: lamarzocco
+ha_platforms:
+  - water_heater
+ha_codeowners:
+  - '@zweckj'
+  - '@rccoleman'
+ha_integration_type: integration
+---
+
+This integration interacts with [La Marzocco coffee machines](https://lamarzocco.com/it/en/) through a mix of cloud calls, local API calls, WebSocket connections and (optionally) Bluetooth.
```
Comment (c0ffeeca7) [inline source/_integrations/lamarzocco.markdown]:
```suggestion
This integration interacts with [La Marzocco coffee machines](https://lamarzocco.com/it/en/) through a mix of cloud calls, local API calls, WebSocket connections, and (optionally) Bluetooth. 
```

File: source/_integrations/lamarzocco.markdown (hunk 1 above)
Comment (zweckj) [reply source/_integrations/lamarzocco.markdown]: yep, added, thanks.

File: source/_integrations/lamarzocco.markdown [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  - '@zweckj'
+  - '@rccoleman'
+ha_integration_type: integration
+---
+
+This integration interacts with [La Marzocco coffee machines](https://lamarzocco.com/it/en/) through a mix of cloud calls, local API calls, WebSocket connections and (optionally) Bluetooth. 
+
+If you are in Bluetooth reach to your coffee machine it should be discovered automatically.
+
+Your machine needs to be added to your account using the official app first.
```
Comment (c0ffeeca7) [inline source/_integrations/lamarzocco.markdown]:
```suggestion
If you are in Bluetooth reach to your coffee machine it should be discovered automatically.

Your machine needs to be added to your account using the official app first.
```
Not sure I understand this correctly. The second sentence here, should it go first? 
First, I have to add the machine to the account in the official app, and then, after I added the integration to Home Assistant, the coffee machine is detected by the integration in Home Assistant?

Comment (zweckj) [reply source/_integrations/lamarzocco.markdown]: It should be discovered anyways. However, to be able to go through the setup you need to it to your account first.

**Review by c0ffeeca7 — DISMISSED** [review]
LGTM. Thank you, @zweckj :+1:

File: source/_integrations/lamarzocco.markdown [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+
+## Sensors
+| Sensor Name | Description | Available for machines | Remarks |
+|-------------|-------------| ---------------------- | ------- |
+| Current coffee temperature | Current temperature of the coffee boiler | all | - |
+| Current steam temperature| Current temperature of the steam boiler | all | - |
+| Drink statistics coffee | Counter for coffees made| all | - |
+| Drink statistics flushing | Counter for total flushes done | all | - |
+| Shot timer | Time the current brew is running | all | Only available when the *Host* was set during component configuration. |
```
Comment (edenhaus) [inline source/_integrations/lamarzocco.markdown]:
```suggestion
```

This PR should only document changes done in https://github.com/home-assistant/core/pull/102291 and not the open ones

**Review by edenhaus — DISMISSED** [review]

File: source/_integrations/lamarzocco.markdown (hunk 4 above)
Comment (zweckj) [reply source/_integrations/lamarzocco.markdown]: Yes, but I can’t add any platforms then until this is merged and I also don’t see why we shouldn’t re-use it while it's still open.

Comment (c0ffeeca7) [reply source/_integrations/lamarzocco.markdown]: > This PR should only document changes done in [home-assistant/core#102291](https://github.com/home-assistant/core/pull/102291) and not the open ones

Thank you for the catch, @edenhaus. I should have seen this.

Comment (zweckj) [reply source/_integrations/lamarzocco.markdown]: @c0ffeeca7 was added after your review, so nothing you didn't catch 😉

File: source/_integrations/lamarzocco.markdown [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+ha_config_flow: true
+ha_domain: lamarzocco
+ha_platforms:
+  - switch
+ha_codeowners:
+  - '@zweckj'
+ha_integration_type: device
+---
+
+This integration interacts with [La Marzocco coffee machines](https://lamarzocco.com/it/en/) through calls to the LaMarzocco cloud API and (optionally) local API calls, which include a WebSocket connections for (near) real-time updates.
```
Comment (c0ffeeca7) [inline source/_integrations/lamarzocco.markdown]:
```suggestion
This integration interacts with [La Marzocco coffee machines](https://lamarzocco.com/it/en/) through calls to the LaMarzocco cloud API and (optionally) local API calls, which include a WebSocket connection for (near) real-time updates. 
```

File: source/_integrations/lamarzocco.markdown [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+ha_platforms:
+  - switch
+ha_codeowners:
+  - '@zweckj'
+ha_integration_type: device
+---
+
+This integration interacts with [La Marzocco coffee machines](https://lamarzocco.com/it/en/) through calls to the LaMarzocco cloud API and (optionally) local API calls, which include a WebSocket connections for (near) real-time updates. 
+
+To be able to configure your machine in Home Assistant, your machine needs to be added to your account using the official La Marzocco app first. Currently, only login with username & password is supported, if you are currently using a social login, you need to create a new LaMarzocco account and transfer your machine to it to be able to use this integration.
```
Comment (c0ffeeca7) [inline source/_integrations/lamarzocco.markdown]:
```suggestion
To be able to configure your machine in Home Assistant, your machine needs to be added to your account using the official La Marzocco app first. Currently, only login with username & password is supported. If you are currently using a social login, you need to create a new LaMarzocco account and transfer your machine to it to be able to use this integration.
```

File: source/_integrations/lamarzocco.markdown [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+  description: "Password you use to log into the La Marzocco app."
+  required: true
+  type: string
+Host:
+  description: "IP address of your machine in your local network. If not set, no local connections will be used."
+  required: false
+  type: boolean
+{% endconfiguration_basic %}
+
+## Switches
```
Comment (c0ffeeca7) [inline source/_integrations/lamarzocco.markdown:39]:
```suggestion
## Switches

```

File: source/_integrations/lamarzocco.markdown [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+| Switch Name | Description | Available for machines |
```
Comment (c0ffeeca7) [inline source/_integrations/lamarzocco.markdown]:
```suggestion
| Switch name | Description | Available for machines |
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @zweckj :+1:

**Pull request merged**